bedrock/settings/static_media.py

```diff
@@ -361,6 +361,14 @@
         ),
         'output_filename': 'css/firefox_firstrun-bundle.css',
     },
+    'firefox_firstrun_membership': {
+        'source_filenames': (
+            'css/base/notification-banner.less',
```

Comment:
Is the CSS here related to the notification here? https://github.com/mozilla/bedrock/pull/5078/files#diff-3d3fd080659e5ac82e85416beb285e4eR80 - If so, do you still need the CSS then?

Reply:
Nice catch 👍